Trustroots/trustroots
# rewrite locations panel to react (first draft so far without using location service)

#### Proposed Changes

* First iteration of migration locations panel (/profile/edit/locations) in the user profile to react - using typeahead component but without querying location service for suggestions

#### Testing Instructions

* Go to /profile/edit/locations
* Try updating "Where do you live" and "Where are you from fields", 
* Make sure new data is persisted
* Check "save" button is enabled/disabled when there are (no) modifications to the fields

Part of https://github.com/Trustroots/trustroots/issues/1334
